include/SDL3/SDL_egl.h

```diff
@@ -25,9 +25,9 @@
  *  This is a simple file to encapsulate the EGL API headers.
  */
 
-#if !defined(_MSC_VER) && !defined(SDL_PLATFORM_ANDROID) && !defined(SDL_USE_BUILTIN_OPENGL_DEFINITIONS)
+#if !defined(_MSC_VER) && !defined(__ANDROID__) && !defined(SDL_USE_BUILTIN_OPENGL_DEFINITIONS)
```

Comment:
Check the history on this? I might have added these `#ifdefs`

Reply:
__ANDROID__ was introduced in 88e3562ba0f481cc25c054b0e52611f2d731b837
    __psp2__/__vita__ was introduced in 134dd4681983bbff18d5aa52ce21974d54f90b39